devices/Pcd8544/Pcd8544.cs

```diff
@@ -0,0 +1,758 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Device.Pwm;
+using System.Threading;
+using Iot.Device.Pcd8544Enums;
+using Iot.Device.CharacterLcd;
+using Iot.Device.Graphics;
+using System.Collections;
+
+namespace Iot.Device
+{
+    /// <summary>
+    /// PCD8544 - 48 × 84 pixels matrix LCD, famous Nokia 5110 screen
```

Comment:
I've seen you've added StyleCop but seems like it is not picking up some of the elements. In this one, the sentence should finish by a dot.

Reply:
Yeah, not sure what is going on here. It gave me some warnings the first time and now it is just quiet. I even re-installed the package. I'm just going to manually do that :)

Reply:
Done!! Turns out StyleCop's C# parser was getting stuck on some of the new C# syntax that it doesn't support yet (maybe ever?) I removed that syntax and it started reporting the warnings again.

Reply:
Great! This version may not support that. It's the time we can move to SDK. Once there, we can use the "modern" version of StyleCop which is supporting the very new part of the language.

Reply:
Is that something that is happening now across the project? Is there something for me to do for this PR?